assets/src/dashboard/app/views/myStories/index.js

```diff
@@ -41,12 +41,31 @@ import NoResults from './noResults';
 
 const FilterContainer = styled.div`
   padding: 0 20px 20px 0;
-  margin: 0 20px;
+  margin: ${({ theme }) => `0 ${theme.pageGutter.desktop}px`};
   border-bottom: ${({ theme: t }) => t.subNavigationBar.border};
+
+  @media ${({ theme }) => theme.breakpoint.smallDisplayPhone} {
+    margin: ${({ theme }) => `0 ${theme.pageGutter.min}px`};
+  }
+
+  @media ${({ theme }) => theme.breakpoint.min} {
+    & > label {
+      border-radius: 0;
+      box-shadow: none;
+      padding: 0 10px 0 0;
+    }
+  }
+`;
+const BodyWrapper = styled.div`
+  margin: ${({ theme }) => `0 ${theme.pageGutter.desktop}px`};
+
+  @media ${({ theme }) => theme.breakpoint.smallDisplayPhone} {
+    margin: ${({ theme }) => `0 ${theme.pageGutter.min}px`};
```

Comment:
i didn't feel like we needed 5 pageGutters, so i just did desktop and min, which is a true min. How do you both feel about intermingling the word 'min' like this?